srcpkgs/LuaJIT/template

```diff
@@ -1,71 +1,58 @@
 # Template file for 'LuaJIT'
 pkgname=LuaJIT
-version=2.1.0beta3
-revision=2
-_so_version=2.1.0
-_dist_version=${_so_version}-beta3
+version=2.1.1706708390
+revision=1
+_commit_hash=9cc2e42b17148036d7d9ef36ab7afe52df345163
+build_style=gnu-makefile
```

Comment:
Could using a version based on `git show` cause problem if the version number goes lower?

Reply:
its not a problem, its using `git show -s --format=%ct` which will return a unix timestamp. Maybe there should be a comment in the template explaining where the version comes from, and how to generate it.

Reply:
Maybe the version should just be "2.1" and not include the timestamp?

As mentioned in my other comment, `.relver` is already included in the tarball.

That way, whenever this package updates, we could just bump the revision, rather than needing to also figure out the unix timestamp

This is just a thought, I don't know if this is the correct or what the preferred way of doing this would be.

Reply:
I don't believe that we should set the version to simply "2.1" and revbump for updates.

For one, it obscures when this is actually updated or when something simply needs to change in the template/shared libs/etc.

For another, when liajit builds and installs, the shared lib has this version, the makefile echoes this as the "minor" version, all the like. By all means, the `git show` *is* the canonical minor version, and it should be reflected in the template